client/landing/stepper/README.md

```diff
@@ -0,0 +1,101 @@
+<img src="https://user-images.githubusercontent.com/17054134/159939643-4a3a7893-ab38-4223-ba5d-5dfe64282f07.png" alt="Stepper logo" width="400">
+
+# Stepper Signup and Onboarding Framework
+The stepper framework is a new framework for quickly spinning up sign up flows. It's meant to make non-linearly easy without sacrificing agility ✨. 
+
+## Non-linearity
+It has been tricky for us to create flows that input-driven steps. Stepper makes it easy by allowing flows to create their own two hooks `useSteps`, and `useStepNavigation`. These hooks have access to the state of of the flow so they can make decisions based on that.
```

Comment:
`create flows that` ... `input-driven steps`: missing something
`to the state of of the flow`: double of